docs/rfcs/0004-suggested-excluded-apps-tech-spec.md

```diff
@@ -0,0 +1,141 @@
+    -   Update the "have seen these apps" settings list with all suggested apps that are on the system. (This means that if a new app is seen on the Settings screen before the daily job runs and we send a notification, we never send a notification. This was confirmed by Santiago.)
+-   (1-2 points) Add recommended section (only if 1 more more apps on the system are suggested) - pull any apps in the "have seen them" list
+-   (1 point) Sort recommended section
+-   (1 point) Sort unrecommended section
+-   (1-2 points) Update sort when something is added or removed
+-   (1-2 points) Ensure search works, shows one section, is alphabetically sorted
+-   (2-5 points) Ensure accessibility is up to date
+-   (1 point) Add metrics for how many suggested (and non-suggested) apps are excluded\
+
+### Day 2
+
+These are important components that we are committed to delivering, but are outside the scope of this epic. They are items that will be done as soon as the epic is closed, but do not block the feature on shipping.
+
+-   Engineering metrics
+    -   Metrics will be created for a successful deep linking, as well as an erroneous deep linking (the app cannot figure out how to resolve the deep link).
+    -   Add performance metrics around load time of the Excluded Apps screen.
+    -   Create a metric for when the daily job was successful, and another one for when it fails.
+    -   A metric should be recorded when the notification is sent
+
+Other options considered
+------------------------
+
+Design and research went through several iterations, so the [designs](https://www.figma.com/file/UZYzma7hlcfE5ke3z8jGbN/App-exclusions-suggestions?type=design&node-id=196-6366&mode=design&t=RL1hdfBQLMS1rKVa-0 "https://www.figma.com/file/UZYzma7hlcfE5ke3z8jGbN/App-exclusions-suggestions?type=design&node-id=196-6366&mode=design&t=RL1hdfBQLMS1rKVa-0") were set before this tech spec was written. (See [completed tickets in the epic](https://mozilla-hub.atlassian.net/browse/VPN-4412 "https://mozilla-hub.atlassian.net/browse/VPN-4412") for more background.)
+
+Other options considered included a fixed list of apps to suggest, which could only be updated when the app updates.
+
+We considered hosting the list in Guardian (which is a private repo, so the list would need to be hosted elsewhere to allow public suggestions) and in the client repo outside of addons (requiring a new app release to update the list).
+
+### Other options considered for suggested app files
+
+The goals for the file format:
+-   Easy to read by humans
+-   Easy for humans to add to, no matter their level of technical proficiency
+-   Easy for us to parse in the app
+
+Initially, there were two additional goals (no longer considerations):
+-   To have one file for all platforms, so that we'd remember to add a specific app to all platforms at once. With a conversation around potentially wanting different suggested app categories on mobile, this seems unimportant.
```

Comment:
Curious why a single file is no longer a goal? Seems desirable but not essential.

Reply:
It seemed important when I thought we were going to exclude the same apps (or types of apps) on all platforms. But it comes at the expense of complexity. And after talking to Santiago, it seems like we may want to suggest excluding very different apps on desktop vs. mobile, so this makes less sense now.

Per a paragraph or two later, I was also a little stuck on how to make volunteer contributions easy, and what format best allowed that. I'm open to suggestions; I don't feel strongly on this one.

That said, I'm curious what comes out of Monday's meeting - we may end up adjusting this based on that discussion.